src/main/flight/mixer_tricopter.c

```diff
@@ -494,7 +418,7 @@ static void triTailTuneStep(servoParam_t *pServoConf, int16_t *pServoVal)
     }
 }
 
-static void tailTuneModeThrustTorque(struct thrustTorque_t *pTT, const bool isThrottleHigh)
+void tailTuneModeThrustTorque(thrustTorque_t *pTT, const bool isThrottleHigh)
```

Comment:
I would suggest to use STATIC_UNIT_TESTED for internal methods that require unit testing. We should hide as much as possible implementation details from the outside world.